content/zh-cn/docs/reference/kubernetes-api/authentication-resources/token-request-v1.md

```diff
@@ -0,0 +1,224 @@
+---
+api_metadata:
+  apiVersion: "authentication.k8s.io/v1"
+  import: "k8s.io/api/authentication/v1"
+  kind: "TokenRequest"
+content_type: "api_reference"
+description: "TokenRequest 为给定的服务账号请求一个令牌。"
+title: "TokenRequest"
+weight: 2
+---
+<!--
+api_metadata:
+  apiVersion: "authentication.k8s.io/v1"
+  import: "k8s.io/api/authentication/v1"
+  kind: "TokenRequest"
+content_type: "api_reference"
+description: "TokenRequest requests a token for a given service account."
+title: "TokenRequest"
+weight: 2
```

Comment:
The `auto_generated: true` line shouldn't been removed.

Reply:
ok, added back.